client/src/index.js

```diff
@@ -5,7 +5,7 @@ const PLAYER = document.querySelector('#player');
 /* eslint-enable no-undef, no-unused-vars */
 
 function setupNetworking() {
-  const serverUrl = process.env.NODE_ENV === 'production' ? 'TODO' : 'http://localhost:3000';
+  const serverUrl = process.env.NODE_ENV === 'production' ? 'http://yummy-server.creativelabsucla.com/' : 'http://localhost:3000';
```

Comment:
```suggestion
  const serverUrl = process.env.NODE_ENV === 'production' ? 'http://chef.creativelabsucla.com/' : 'http://localhost:3000';
```

Change to this?

Reply:
should we do this in a chore? when we change everything in the repo to chef?

Reply:
yeah sounds good, its not a big deal anyway